dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.mpFaultTolerance2.0-cdi1.2.feature

```diff
@@ -1,10 +1,10 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=com.ibm.websphere.appserver.mpFaultTolerance2.0-cdi
+symbolicName=com.ibm.websphere.appserver.mpFaultTolerance2.0-cdi1.2
 visibility=private
 singleton=true
 IBM-Provision-Capability: \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.mpFaultTolerance-2.0))", \
-  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.cdi-1.2)(osgi.identity=com.ibm.websphere.appserver.cdi-2.0)))"
+  osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.cdi-1.2))"
 -bundles=com.ibm.ws.microprofile.faulttolerance.cdi,\
   com.ibm.ws.microprofile.faulttolerance.cdi.2.0.services
```

Comment:
This bundle refers to FT 2.0 so it should be named
`com.ibm.ws.microprofile.faulttolerance.2.0.cdi.services`
otherwise is appears to refer to any version of FT but only CDI 2.0

Reply:
Raised #6182 to cover this as changing the bundle names will disrupt ongoing work.